packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors-for-charts/colors-grid.js

```diff
@@ -12,7 +12,7 @@ const ColorsGrid = () => (
         <GridItem span={10}>
           <div className="ws-content-chart-colors-gallery--information">
             <div>#8BC1F7</div>
```

Comment:
This is going to change between light/dark theme. I'm not sure what a good fix is. I don't think we have tokens currently for dark theme values - either a dark theme value in the current/existing token, or potentially a separate token for dark theme - but if we did, this whole grid bit could probably just loop through the chart color tokens and spit out the var/hex color from the token, and include the dark theme hex color and hide it in light theme and show it in dark theme?